src/plugins/vis_type_xy/public/components/split_chart_warning.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { FC } from 'react';
+
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiPageContentBody,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export const SplitChartWarning: FC = () => {
+  return (
+    <EuiPage style={{ width: '100%' }}>
+      <EuiPageBody component="div">
+        <EuiPageContent verticalPosition="center" horizontalPosition="center">
+          <EuiPageContentHeader>
+            <EuiPageContentHeaderSection>
+              <EuiTitle>
+                <h2>
+                  {i18n.translate('visTypeXy.splitChartWarning.title', {
+                    defaultMessage: 'Warning',
+                  })}
+                </h2>
+              </EuiTitle>
+            </EuiPageContentHeaderSection>
+          </EuiPageContentHeader>
+          <EuiPageContentBody>
+            {i18n.translate('visTypeXy.splitChartWarning.content', {
+              defaultMessage:
+                'Warning split chart aggregations are not supported with vis_type_xy plugin',
```

Comment:
That's a good thought @stratoula, let me see if I can disable that from the editor level via the advanced settting.

Reply:
I added `disabled` and `tooltip` to the defaultEditor schema to be use on the agg add component. I also added a default tooltip for values disabled via max prop, only shown when disabled. Starting on this https://github.com/elastic/kibana/issues/43897

See 2c4d351822dc4bdb10a634ffc8917555d4f04323

![Screen Recording 2020-11-05 at 04 25 PM](https://user-images.githubusercontent.com/19007109/98303542-dfbfd500-1f83-11eb-8027-ea488ada4a02.gif)

Reply:
I refactored (4a91edbac00305508104ceb7468e0798667c0889) the `SplitChartWarning` to use an `EuiCallOut` for preexisting visualizations with split series aggs that are opened with the new `vis_type_xy` plugin. The chart will render no data ui.

![Screen Recording 2020-11-05 at 05 37 PM](https://user-images.githubusercontent.com/19007109/98308508-d2a7e380-1f8d-11eb-9495-048544ea8f74.gif)

For some reason these changes (i.e. putting `SplitChartWarning` in the `vis_renderer.tsx`) I get the error below. Any ideas?

```
[React Intl] Could not find required `intl` object. <IntlProvider> needs to exist in the component ancestry. Using default message as fallback. 
    at FormattedMessage (http://localhost:5901/9007199254740991/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:328593:5)
    at div
    at EuiText (http://localhost:5901/9007199254740991/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.@elastic.js:128198:24)
    at div
    at EuiCallOut (http://localhost:5901/9007199254740991/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.@elastic.js:44270:20)
    at SplitChartWarning
```

Reply:
Try to add `<I18nProvider` as I have added it here https://github.com/elastic/kibana/pull/82810/files#diff-87273e7ac8d278b4f23f1e762b80abe0d0fcbca18e5cfe99345e1b36fd00336bR28. I think this will solve the problem

Reply:
Perfect, Thanks!